autofix-printing-stats/run

```diff
@@ -204,15 +212,18 @@ let upload_results should_upload results =
              ];
            print_newline ()))
 
+let make_opts () =
```

Comment:
Using ppx_deriving_cmdliner like in hello_script.ml would remove the need for this code, but this is fine also like this.

Reply:
For something this simple, it seemed easier to just roll my own rather than learn another tool and then make sure it's available in CI.

Reply:
I agree